spec/helpers/time_options_helper_spec.rb

```diff
@@ -49,6 +49,13 @@
     end
   end
 
+  describe "#format_full_date" do
+    it "formats the date correctly" do
+      date = Time.new(2030, 4, 20, 10, 26, 0, '+01:00') # London timezone
```

Comment:
Can probably simplify to:

```suggestion
      date = Time.new(2030, 4, 20)
```

As we're only concerned with the date. Should at least be able to remove the London timezone comment and `+01:00` modifier 👍